gau2grid/RSH.py

```diff
@@ -43,7 +43,7 @@ def _load_saved_rsh_coefs():
 
 
 # Windows does not support caching due to missing numpy.float128
-if platform.system() in ['Linux', 'Darwin', 'Windows']:
+if platform.system() in ['Linux', 'FreeBSD', 'Darwin', 'Windows']:
```

Comment:
I might have been hasty, if `platform.system()` did not evaluate here to `Linux` this would pre-empt the below patch. Can you check if only changing line 46 and leaving line 169 unchanged fixes the issue?